src/Sulu/Bundle/SecurityBundle/Resources/public/js/services/role-manager.js

```diff
@@ -0,0 +1,186 @@
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define([
+    'services/husky/util',
+    'services/husky/mediator',
+    'sulusecurity/models/role'
+], function(
+    Util,
+    Mediator,
+    Role
+) {
+
+    'use strict';
+
+    var instance = null,
+
+        /**
+         * Saves a role with given data
+         *
+         * @param {Object} data The data to save
+         * @returns {Object} a promise
+         */
+        saveRole = function(data) {
```

Comment:
Would call that only `save`, since `roleManager.save` already says that a role is being saved